ui-components/contextform.md

```diff
@@ -49,49 +49,49 @@ This relates to the form itself. The form specifications are:
 
 {% include context/positioning.md %}
 
-### Context form buttons
+## Context form buttons
 
 Unlike normal context toolbar buttons, Context form buttons are not registered beforehand. Instead, you need to define each button completely in the `commands` section.
 
-#### contextformbutton
+### Context form button
```

Comment:
The title was inconsistent with the below ` Context form toggle button` heading.